wiki/markdown/Add-Subscription.md

```diff
@@ -0,0 +1,45 @@
+# Add Subscription
+
+This form is used to add subscription through frontend GUI. Here is a 
+description of the elements available on this form.
+
+**Load Subscription Template:** In this dropdown list options are given to 
+select a specific template for the subscription form. Currently, the following 
+options are listed: Jenkins Pipeline Parametrized Job Trigger, REST POST and 
+Mail Trigger.
+
+**SubscriptionName:** A field to give a name to the current subscription. Only 
+letters, underscore and numbers are allowed in this field.
+
+**NotificationType:** A dropdown list with two options, i.e., REST POST and 
+Mail Trigger, depending on how a subscriber want to be notified when a 
+subscription is fulfilled.
```

Comment:
If Notification type MAIL is selected an email subject may be entered, but it is optional.

Reply:
Done